util/chplenv/utils.py

```diff
@@ -181,6 +181,23 @@ def is_ver_in_range(versionStr, minimumStr, maximumStr):
 
     return False
 
+def check_valid_var(varname, value, valid_values):
+    """Check that a variable is set to a valid value"""
```

Comment:
Would be nice to say here that valid_values can be a tuple or a list and that 'varname' is only used for the error message.